x-pack/test/functional/services/ml/api.ts

```diff
@@ -455,10 +455,27 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       });
     },
 
+    validateJobId(jobId: string) {
+      if (jobId.match(/[\*,]/) !== null) {
+        throw new Error(`No wildcards or list of ids supported in this context (got ${jobId})`);
+      }
+    },
+
     async getAnomalyDetectionJob(jobId: string) {
+      this.validateJobId(jobId);
```

Comment:
`getAnomalyDetectionJob` possibly doesn't need this check. We may in the future wish to get wild carded or multiple jobs.
The real issue has been addressed in the `adJobExist` function which wouldn't work as expected with wildcards or commas separated IDs

Reply:
Addressed in b7bf8dd .